Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java

```diff
@@ -562,7 +563,9 @@ public void testBlockingLockWithinWorkflows() {
 		       	lockHandle.setLockName(decisionApi.getContextValue(orderUri+"#0", "lockName"));	
 	       	}
         }
-        Assert.assertTrue(lockApi.releaseLock(lockUri.toString(), lockConfig.getName(), lockHandle));
+        try {
```

Comment:
I believe Dave is correct in his description of how releaseLock works. Should i file a ticket to change it to not throw an exception (and also not release lock) ?